applications/wg-easy/Taskfile.yaml

```diff
@@ -519,3 +528,99 @@ tasks:
       - task: helm-install
       - task: test
       - task: cluster-delete
+
+  cmx-vm-create:
+    desc: Create a CMX VM instance using Replicated CLI
+    run: once
+    silent: false
+    status:
+      - |
+        # Check if VM is running
+        replicated vm ls | grep "{{.CMX_VM_NAME}}" | grep running
+    cmds:
+      - |
+        echo "Creating CMX VM {{.CMX_VM_NAME}}..."
+        replicated vm create --distribution {{.CMX_VM_DISTRIBUTION}} --version {{.CMX_VM_VERSION}} --instance-type {{.CMX_VM_INSTANCE_TYPE}} --disk {{.CMX_VM_DISK_SIZE}} --name {{.CMX_VM_NAME}} --ttl {{.CMX_VM_TTL}}
+
+        echo "Waiting for VM to be running (timeout: 120s)..."
+        for i in $(seq 1 60); do
+          replicated vm ls | grep "{{.CMX_VM_NAME}}"
+          if replicated vm ls | grep "{{.CMX_VM_NAME}}" | grep running; then
+            echo "VM {{.CMX_VM_NAME}} is ready!"
+            break
+          fi
+          if [ $i -eq 60 ]; then
+            echo "Timeout reached after 120s. VM may still be initializing."
+            exit 1
+          fi
+          sleep 2
+        done
+
+  cmx-vm-delete:
+    desc: Delete a CMX VM instance
+    silent: false
+    cmds:
+      - |
+        echo "Deleting CMX VM {{.CMX_VM_NAME}}..."
+        replicated vm rm {{.CMX_VM_NAME}}
+
+  cmx-vm-install:
+    desc: Download and install the app as Embedded Cluster on CMX VM
+    silent: false
```

Comment:
Would be good to silence this. Logs from the command are quite noisy by themselves.